generators/typescript-multiple-files/model.handlebars

```diff
@@ -1,3 +1,4 @@
+/* tslint:disable */
```

Comment:
Why do we need to add this line? the linter should not be run on `handlebars` file